### ISSUE TITLE
Advise needed: How to secure rancher driven application itself so that it can ONLY be accessed thru rap?

### ISSUE THREAD

**Issue by hanselke** [adi90x/rancher-active-proxy#43]
So this is more of a rancher question, but i wish to restrict access to all applications whichcurrently have domain names driven by rap so that it only ever goes thru the nginx-proxy.

Purpose is to be able to install nginx-lua into rap, so that i'm able to run lua scripts against every connection.

**Comment by adi90x** [comment]
If there is no external mapped port for containers they will only be accessible via RAP no ?

**Comment by hanselke** [comment]
I dont think so. when there is no external port for container, i think RAP just doesnt do anything

hansel

**Comment by adi90x** [comment]
No external port for RAP containers or the others ?
As main goal of RAP, is to route trafic from one input port(80/http most of
the time) to several containers not exposing a port on the outside of
rancher, lets say you have RAP exposing & listening on port 80 of host,
then you can have unlimited number of Web containers exposing 80 but not
listening on host port  and RAP will route input trafic based on hostname
Is it clearer ?
